kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/ProcessResource.java

```diff
@@ -199,6 +200,52 @@ public Response startProcess(@javax.ws.rs.core.Context HttpHeaders headers,
         }
     }
 
+
+    @ApiOperation(value="Starts a new synchronous process instance of a specified process.")
+    @ApiResponses(value = { 
+            @ApiResponse(code = 201, response=Long.class, message = "Process instance started", examples=@Example(value= {
```

Comment:
Still missing the response class as Map 
```suggestion
            @ApiResponse(code = 201, response=Map.class, message = "Process instance started", examples=@Example(value= {
```